### PR TITLE
pr feedback

### PR DESCRIPTION
**Pull request by lingpri**
addressing PR feedback.

https://github.com/Graylog2/documentation/pull/976/files#r516096010
https://github.com/Graylog2/documentation/pull/976/files#r516229278

**Screenshot**

slack_pr_feedback

### CODE REVIEW

File: pages/alerts.rst [hunk 1]
```diff
@@ -304,13 +304,10 @@ Configuration Color
...
 
 Custom Message
-    A custom message to be appended below the alert title.
+    This is a message template and the data described above will be appended below the alert title, when the notification is sent.
 
 User Name (optional)
-    Addressing a specific `user  <https://api.slack.com/reference/surfaces/formatting#mentioning-users>`__ in the slack workspace.
-
-Icon URL
-    Image to use as the icon for this message.
```
Comment (waab76) [inline pages/alerts.rst]: Did you mean to delete Icon URL?

File: pages/alerts.rst [hunk 2]
```diff
@@ -304,13 +304,10 @@ Configuration Color
     Highlight the custom message with this `color <https://api.slack.com/reference/surfaces/formatting#quotes>`__ .
 
 Custom Message
-    A custom message to be appended below the alert title.
+    This is a message template and the data described above will be appended below the alert title, when the notification is sent.
```
Comment (waab76) [inline pages/alerts.rst]:
```suggestion
    The message that will be sent to Slack.  The data described above can be used in this template.
```

**Review by waab76 — CHANGES_REQUESTED** [review]

File: pages/alerts.rst (hunk 1 above)
Comment (lingpri) [reply pages/alerts.rst]: good catch.

**Review by waab76 — APPROVED** [review]
LGTM

**Pull request merged**